### PR TITLE
Make examples more clear

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
I've seen some [NEW CONTENT TYPE REQUEST] issues that misunderstand the template. People think that the bullet points are part of one big example of a list. They are intended as a list of individual examples. I thought that the quotation marks, and entire comma separated lists under one bullet point made that clear when I wrote it. You don't need to use bullet points for one item, or keep the word "Examples", like [this](https://github.com/google/magika/issues/291), or [this](https://github.com/google/magika/issues/269), or [this](https://github.com/google/magika/issues/240), or [this](https://github.com/google/magika/issues/230), or [this](https://github.com/google/magika/issues/229), or [this](https://github.com/google/magika/issues/228), or, [this](https://github.com/google/magika/issues/227), or [this](https://github.com/google/magika/issues/218), or, [this](https://github.com/google/magika/issues/219), or [this](https://github.com/google/magika/issues/120), or [this](https://github.com/google/magika/issues/106), or [this](https://github.com/google/magika/issues/83), or [this](https://github.com/google/magika/issues/240).

### CODE REVIEW

File: docs/js.md [hunk 1]
```diff
@@ -206,26 +206,26 @@ Returns **[Promise][21]\<ModelResult>** A dictionary containing the top label an
 
 [18]: #parameters-7
 
-[19]: https://github.com/google/magika/blob/48d4394825d19221b6413203469a549b15a841ae/js/magika_node.ts#L29-L134 "Source code on GitHub"
+[19]: https://github.com/MichaelHinrichs/magika/blob/ce2e897198693c9c07c5e0ae72ea2a5a4d03d96e/js/magika_node.ts#L29-L134 "Source code on GitHub"
```
Comment (gaby) [inline docs/js.md:209]: @MichaelHinrichs This should be changed back to `github.com/google`

**Review by gaby — CHANGES_REQUESTED** [review]

File: docs/js.md (hunk 1 above)
Comment (MichaelHinrichs) [reply docs/js.md:209]: I don't know how it got like that. I never touched that file.

Comment (MichaelHinrichs) [reply docs/js.md]: Should i change the folder between "blob" and "js" back too?

**Comment by reyammer** [comment]
Hello folks! sorry for the delay in replying, we are busy working on the next big release :-) In the meantime, it's good to improve the GH template, but what's about the changes in docs/js? can we remove changes to that file from the PR so that we can focus on improving the GH template? Thanks!

**Comment by MichaelHinrichs** [comment]
I'll take that as a "yes" to my question in the above conversation. Changing it back now.

**Pull request merged**